docs/docs/volumes/cache.md

```diff
@@ -70,14 +76,40 @@ $ wget https://github.com/dragonflyoss/Dragonfly2/releases/download/v2.1.44/dfge
 $ rpm -ivh dfget-2.1.44-linux-amd64.rpm
 ```
 
-6. Cancel the cluster default configuration sc
+6. In order not to affect the subsequent, we need to cancel the previously configured default sc
+```console
+ $ kubectl patch storageclass hwameistor-storage-lvm-hdd -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
```

Comment:
```suggestion
kubectl patch storageclass hwameistor-storage-lvm-hdd -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
```
If only a command, `$` is not necessary, which may add extra action to delete it.